<ui version="4.0" >
 <class>VCXYPadProperties</class>
 <widget class="QDialog" name="VCXYPadProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>XY Pad Properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="m_horizontalGroup" >
     <property name="title" >
      <string>Horizontal axis (X)</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="m_listX" >
        <column>
         <property name="text" >
          <string>Fixture</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Channel</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Min</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Max</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Reverse</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QToolButton" name="m_addX" >
          <property name="toolTip" >
           <string>Add a channel to this pad's X axis</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="main.qrc" >
            <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_removeX" >
          <property name="toolTip" >
           <string>Remove the selected channel from this pad's X axis</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="main.qrc" >
            <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="m_minXLabel" >
          <property name="text" >
           <string>Minimum value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_minXSpin" >
          <property name="toolTip" >
           <string>Lowest value for the selected channel on the X axis</string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_maxXLabel" >
          <property name="text" >
           <string>Maximum value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_maxXSpin" >
          <property name="toolTip" >
           <string>Highest value for the selected channel on the X axis</string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
          <property name="value" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_reverseX" >
          <property name="toolTip" >
           <string>Reverse the selected channel's X axis</string>
          </property>
          <property name="text" >
           <string>Reverse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_verticalGroup" >
     <property name="title" >
      <string>Vertical axis (Y)</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="m_listY" >
        <column>
         <property name="text" >
          <string>Fixture</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Channel</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Min</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Max</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Reverse</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QToolButton" name="m_addY" >
          <property name="toolTip" >
           <string>Add a channel to this pad's Y axis</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="main.qrc" >
            <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_removeY" >
          <property name="toolTip" >
           <string>Remove the selected channel from this pad's Y axis</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="main.qrc" >
            <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="m_minYLabel" >
          <property name="text" >
           <string>Minimum value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_minYSpin" >
          <property name="toolTip" >
           <string>Lowest value for the selected channel on the Y axis</string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_maxYLabel" >
          <property name="text" >
           <string>Maximum value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_maxYSpin" >
          <property name="toolTip" >
           <string>Highest value for the selected channel on the Y axis</string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
          <property name="value" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_reverseY" >
          <property name="toolTip" >
           <string>Reverse the selected channel's Y axis</string>
          </property>
          <property name="text" >
           <string>Reverse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VCXYPadProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VCXYPadProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
